elements/pfe-tabs/BaseTab.ts

```diff
@@ -23,11 +23,9 @@ export abstract class BaseTab extends LitElement {
   @queryAssignedElements({ slot: 'icon', flatten: true })
   private icons!: Array<HTMLElement>;
 
-  @observed
-  @property({ reflect: true, type: Boolean }) active = false;
+  abstract active: boolean;
 
-  @observed
-  @property({ reflect: true, type: Boolean }) disabled = false;
+  abstract disabled: boolean;
 
   #internals = this.attachInternals();
```

Comment:
out of scope for this PR, but it could be fun to move this to internals controller or a subclass thereof, then apply those to the subclasses